keyboards/handwired/onekey/keymaps/digitizer/config.h

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file should be deleted.

Reply:
Yeah that makes sense obviously, oversight on my part too.